### ISSUE TITLE
Me systems working as chunkloaders

### ISSUE THREAD

**Issue by 123Kappa** [AppliedEnergistics/Applied-Energistics-2#3960]
**Describe the bug**
I am hosting a project ozone 3 server and my server was lagging. I noticed that the server had too many Tile entities, after further investigation I realized that some chunks are chunkloaded so I started teleporting to a chunkloaded base, removing an item and then went back to spawn. 
And then i finally found what chunkloaded the base It was a small me system. 

**To Reproduce**
The me system looked like this: https://imgur.com/9PJjr4I . I removed one of each component (chest, cable or acceptor) and it chunkloaded only when all three where connected to each other.
Then I tried it with a different simple system, this one: https://imgur.com/a/m4FIR3l 
and only when all three were connected it chunkloaded. 

So basically me systems chunkload.

Note: When the server starts nothing is chunkloaded you have to go to the chunk at least one time and it will stay chunkloaded forever, also the systems do not need power. 

**Expected behavior**
Me system not getting chunkloaded. 

**Environment**
- Minecraft Version: 1.12.2
- AE2 Version: rv6-stable-6
- Forge Version: forge-1.12.2-14.23.5.2808
- Also the modpack has three addons:
          - ae2stuff-0.7.0.4-mc1.12.2
          - WirelessCraftingTerminal-1.12.2-3.11.88
          - ExtraCells-1.12.2-2.6.2a

**Comment by yueh** [comment]
Duplicate of #3957
